{"name":"Lesson repository create lesson failure","fullName":"TestLessonCreateSuite/Lesson repository create lesson/TestCreate_Failure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727875153113,"stop":1727875153113,"uuid":"eaae1ff6-80c0-11ef-9e41-ca15b1947337","historyId":"302356b2f059cca0aea0004d53fa7880","testCaseId":"ea9457d9ae31e4848edf6aaf4acbd416","labels":[{"name":"language","value":"go1.22.5"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"macbook.local"},{"name":"thread","value":"TestLessonCreateSuite/Lesson repository create lesson/TestCreate_Failure"},{"name":"suite","value":"Lesson repository create lesson"},{"name":"package","value":"github.com/paw1a/eschool/internal/adapter/repository/postgres/test"}],"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1727875153113,"stop":1727875153113,"parameters":[{"name":"Error","value":"sql: connection is already closed: persistence internal error"},{"name":"Target","value":"persistence internal error"}]}]}